posthog/clickhouse/test/__snapshots__/test_schema.ambr

```diff
@@ -2175,7 +2177,11 @@
       distinct_id VARCHAR,
       min_first_timestamp SimpleAggregateFunction(min, DateTime64(6, 'UTC')),
       max_last_timestamp SimpleAggregateFunction(max, DateTime64(6, 'UTC')),
+      -- store the first url of the session so we can quickly show that in playlists
       first_url AggregateFunction(argMin, Nullable(VARCHAR), DateTime64(6, 'UTC')),
+      -- but also store each url so we can query by visited page without having to scan all events
+      -- despite the name we can put mobile screens in here as well to give same functionality across platforms
+      all_urls SimpleAggregateFunction(groupUniqArrayArray, Array(String)),
```

Comment:
@robbie-c I realised after figuring this out that the sessions table has a URLs field too
but we've ended up with different solutions... 
is it just different inputs, am I missing a trick from something you discovered while making raw_sessions?

(i'm trying to reduce dependency on events for "basic" functionality)

Reply:
Mine is also a `urls SimpleAggregateFunction(groupUniqArrayArray, Array(String)),`! Unless there's another difference you're referring to.

I don't have my own ingester though, it's coming straight from events one at a time so I just have to use what's there.

Reply:
Ah, cool, i saw some array merge, array flatten that i'm not doing 

all good 👍 

(was lazy of me not to link)

Reply:
https://github.com/PostHog/posthog/blob/fa716372c30050795df49cd2e6d33611c9bb5570/posthog/hogql/database/schema/sessions_v2.py#L198

Reply:
hmmm great context thank you... i was hoping this was doing that for free for me... will check this before i roll out anything built on top of it!

Reply:
It still does a lot, most of the time! It's a big saving on bytes read, but it's just not guaranteed to happen 🙃